index.html

```diff
@@ -39,22 +39,276 @@ <h1 class="main-text font-poppins">
+          </section>
+        </article>
+        <!---->
+        <article class="card">
+          <figure>
+            <img
+              class="img-card"
+              src="images/Snapshoot Portfolio (4).svg"
+              alt=""
+            />
+          </figure>
+
+          <section class="card-description">
+            <header class="header-card">
+              <h1>Tonic</h1>
+              <ul>
+                <li>Canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </header>
+
+            <div class="paragraph-card">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+
+            <div>
+              <ul class="languages-card">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+            </div>
+
+            <div>
+              <button class="btn">See Project</button>
+            </div>
+          </section>
+        </article>
+      </section>
+    </section>
+    <!--About-->
+    <article class="about-section">
+      <section class="hero">
+        <h1 class="main-text">About<br />Myself</h1>
+        <p class="main-paragraph">
+          Hello! I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <p class="main-connect font-poppins">LET’S CONNECT</p>
+        <!--Social Media-->
+        <section class="social-media">
+          <ul>
+            <li>
+              <a href="https://twitter.com/_JavierAybar">
+                <img
+                  class="social-media-icon"
+                  src="images/Twitter-icon.svg"
+                  alt="Twitter"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/javier-aybar-932376274/">
+                <img
+                  class="social-media-icon"
+                  src="images/Linkedin icon.svg"
+                  alt="Linkedin"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="https://github.com/JavierAybar">
+                <img
+                  class="social-media-icon"
+                  src="images/Github-icon.svg"
+                  alt="GitHub"
+                />
+              </a>
+            </li>
+          </ul>
+        </section>
+        <div>
+          <button class="btn">Get my resume</button>
+        </div>
+      </section>
+    </article>
+    <!--Languages-->
+    <section class="languages-flex">
+      <ul class="languages">
+        <ul>
+          <li class="active-dropdown"> 
+            <p>Languages</p>
+          </li>
+
```

Comment:
good job by using ul for list of items but an arrow icon is missing here 
![image (20)](https://github.com/JavierAybar/Portfolio/assets/59707859/ab7bd4f6-627e-4463-9a98-fceea48c561b)
![image (21)](https://github.com/JavierAybar/Portfolio/assets/59707859/7a029b4f-3cc9-4198-a694-9a13305242ac)

Reply:
In this section you talk about the icon like this right? (>)